cloudfour/cloudfour.com-patterns
# Chore: style Gutenberg file block

## Overview

See #710

This PR introduces styles for Gutenberg file blocks. File blocks can be displayed in one of two ways, as a "download link" or as a "download button". 

The changes being introduced here, style both links and buttons with primary, secondary and tertiary button styles. 

## Screenshots

Example of the WP Gutenberg (file) block editor interface, demonstrating "download links" and "download buttons"

![Screen Shot 2020-06-03 at 4 10 40 PM](https://user-images.githubusercontent.com/1427548/83697835-d2598580-a5b4-11ea-90ab-c352ecd2b69e.png)

Styled WordPress output

![file-hover](https://user-images.githubusercontent.com/1427548/83691472-13e33400-a5a7-11ea-9f09-10e7b3b7f732.gif)

Example of Storybook documentation

![Screen Shot 2020-06-03 at 4 37 41 PM](https://user-images.githubusercontent.com/1427548/83699350-a5a76d00-a5b8-11ea-9553-42f275ac8bcd.png)


## Testing

1. View [deploy preview](https://deploy-preview-773--cloudfour-patterns.netlify.app/?path=/docs/vendor-wordpress-gutenberg--core-file)
1. Confirm links in story description are linked correctly
1. Review CSS 
1. Confirm active, focus and Hover are working as expected

## Issue

See #710
